helm-charts/sep-service/templates/README.md

```diff
@@ -61,9 +61,75 @@ metadata:
 type: Opaque
```

Comment:
all files in templates get processed by helm install. moving readme into templates directory broke your deployment of the sep service.   i pushed an empty commit to kick off the deployment again. here you can see the failure of helm trying to process it
https://buildmeister-v3.stellar-ops.com/job/ProductEngineering/job/stellar-anchor-platform-preview/956/

Reply:
Thanks for catching, it's been restored to its original location